recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@ requirements:
   host:
     - python >=3.8
     - pip
-    - poetry
+    - poetry <1.6
```

Comment:
Same deal: this should be `poetry-core >=1.0.0`:
https://github.com/supabase-community/supabase-py/blob/v1.2.0/pyproject.toml#L51